packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -251,6 +253,8 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
   let gridCellProps: DOMAttributes = mergeProps(itemProps, {
     role: 'gridcell',
     onKeyDownCapture,
+    'aria-colspan': node.colspan,
```

Comment:
can `colspan` match the capitalisation of the GridCellProps' `colSpan`?

Reply:
It should be a fairly straightforward to change, but I think it would introduce a breaking change in table header (`useTableColumnHeader`): https://react-spectrum.adobe.com/react-aria/useTable.html#table-header

Reply:
Ah, then probably we should go the other direction to match.

Reply:
Ok, spent a little longer looking at this, I think we can match the `colSpan` if we keep `colspan` on the node at the same time, and we can deprecate the old one. Then the hooks should return the appropriate values so that people don't need to do that step with hooks themselves

Reply:
I have renamed it now and duplicated the assignments to ensure backward compatibility.